rundeckapp/grails-app/views/scheduledExecution/_edit.gsp

```diff
@@ -947,5 +947,5 @@ function getCurSEID(){
 <!--[if (gt IE 8)|!(IE)]><!--> <asset:javascript src="ace/ext-language_tools.js"/><!--<![endif]-->
 <div id="msg"></div>
 
-    <g:render template="/framework/storageBrowseModalKO"/>
+<g:render template="/framework/storageBrowseModalKO"/>
```

Comment:
the reformatting on this file makes it impossible to tell what was changed. is it possible to reset the formatting so that actual changes can be seen in the diff?